crates/uv/src/main.rs

```diff
@@ -1012,10 +1016,26 @@ async fn run() -> Result<ExitStatus> {
                 Vec::new(),
                 args.no_index,
             );
+            let cwd_save: PathBuf;
+            let prompt = if args.prompt == "." {
+                match env::current_dir() {
+                    Ok(cwd) => {
+                        cwd_save = cwd.clone();
+                        match cwd_save.file_name() {
+                            Some(b) => b.to_str(),
+                            None => None,
+                        }
+                    }
+                    Err(_) => None,
+                }
+            } else {
+                Some(args.prompt.as_str())
+            };
```

Comment:
I think this logic should reside in `gourgeist::create_bare_venv` so that the behavior remains the same through the API and CLI similar to how the [`venv.EnvBuilder`](https://docs.python.org/3/library/venv.html#venv.EnvBuilder) does in standard library.

Reply:
I moved "." to basename(cwd) logic to create_bare_venv.